### PR TITLE
Create url-check.yml

### LINKED ISSUE #126: check URL links in CI
Check in CI that URL links are valid.

### PR DESCRIPTION
**Pull request by MarcoIeni**
Attempt to fix #126

### CODE REVIEW

File: .github/workflows/url-check.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: Check Markdown links
+
+on: push
```
Comment (MarcoIeni) [inline .github/workflows/url-check.yml:3]:
```suggestion
on:
  push:
    branches: [master]
  pull_request:
```

**Comment by MarcoIeni** [comment]
I expected this to run. I will try this in my fork.

**Comment by simonsan** [comment]
Nice! Why is crates.io returning `404` for pages that exist though 🤔

**Comment by simonsan** [comment]
👍🏽

**Comment by MarcoIeni** [comment]
I think we can do better.
On PR and push we check for modified files.
Periodically we check all files.

**Comment by simonsan** [comment]
> I think we can do better.
> On PR and push we check for modified files.
> Periodically we check all files.

Sounds good

**Review by simonsan — APPROVED** [review]

**Pull request merged**